qcodes_contrib_drivers/drivers/Tektronix/Keithley_2000_Scan.py

```diff
@@ -0,0 +1,65 @@
+from functools import partial
+from typing import TYPE_CHECKING
+
+from qcodes.instrument.channel import InstrumentChannel
```

Comment:
The previous import was the correct import according to qcodes public API.
I deleted my comment in this regard right after making that comment, but you changed it anyway. So please revert back to the previous import in both files.